dash/development/_r_components_generation.py

```diff
@@ -625,3 +641,74 @@ def generate_exports(
         package_imports,
         package_suggests,
     )
+
+
+def get_r_prop_types(type_object):
+    """Mapping from the PropTypes js type object to the R type"""
+
+    return dict(
+        array=lambda: "unnamed list",
+        bool=lambda: "logical",
+        number=lambda: "numeric",
+        string=lambda: "character",
+        object=lambda: "named list",
+        any=lambda: 'logical | numeric | character | named list | unnamed list',  # noqa:E501
+        element=lambda: "dash component",
+        node=lambda: "a list of or a singular dash " "component, string or number",  # noqa:E501
```

Comment:
If this gets collapsed to 1 line (which I guess it can because black uses longer lines than pep8), smoosh it back into one string - ie just `...dash component...`, not `...dash " "component...`

Reply:
and again, if we're going to use black we should just use black... tell flake8 to accept a line length of 88 and get rid of these `# noqa` directives.

Reply:
Good point. I don't have a preference here. I can correct the reformatted lines until we discuss using `black` for Dash as well.

Reply:
fixed in 2eb3a070adfd5b913e39d9981d951669b4c3814c